File: GeoCode/GeoCode.Models/Address.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;
using Microsoft.EntityFrameworkCore;

namespace GeoCode.Models
{
    public class Address
    {
        public string Id { get; set; }
        public string AddressString { get; set; }
        public double Latitude { get; set; }
        public double Longitude { get; set; }
        public int NumberOfHits { get; set; }
    }
}

File: GeoCodeClient/src/app/models/address.ts
export class Address {
  addressString: string;
  latitude: number;
  longitude: number;
  numberOfHits:number;

  constructor(address:string, latitude:number, longitude:number, numberOfHits:number) {
    this.addressString=address;
    this.latitude=latitude;
    this.longitude=longitude;
    this.numberOfHits=numberOfHits;
  }
}
